src/Compilers/Core/AnalyzerDriver/AnalyzerExecutor.cs

```diff
@@ -495,8 +495,9 @@ internal static bool IsCanceled(Exception ex, CancellationToken cancellationToke
         internal static Diagnostic GetAnalyzerExceptionDiagnostic(DiagnosticAnalyzer analyzer, Exception e)
         {
             var descriptor = new DiagnosticDescriptor(AnalyzerExceptionDiagnosticId,
-                AnalyzerDriverResources.AnalyzerFailure,
-                AnalyzerDriverResources.AnalyzerThrows,
+                title: AnalyzerDriverResources.AnalyzerFailure,
+                messageFormat: AnalyzerDriverResources.AnalyzerThrows,
+                description: string.Format(AnalyzerDriverResources.AnalyzerThrowsDescription, e.ToString()),
```

Comment:
Do we need the additional header text (Analyzer threw an exception with stack trace: ) before the stack trace here? That information is already present in the message (i.e. in the non-expanding portion of the error).

If we want a header. It may be better to just repeat whatever shows up in the primary message (i.e. Analyzer '{0}' threw an exception with message '{1}')...